cmd/neofs-node/metrics.go

```diff
@@ -25,3 +31,34 @@ func initMetrics(c *cfg) *httputil.Server {
 
 	return srv
 }
+
+func (c *cfg) setShardsCapacity() error {
```

Comment:
You obviously can share the result of `writeSystemAttributes` with this. Or at least refactor out some common code. DRY.